src/App.js

```diff
@@ -1,19 +1,47 @@
-import React, { Component } from "react";
+import React, { Component } from 'react'
+import Heading from './Heading'
+import TouristInfoCards from './TouristInfoCards'
+import Footer from './Footer'
+import SearchResults from './SearchResults'
 
-import Bookings from "./Bookings";
-import Restaurant from "./Restaurant";
-import "./App.css";
+import Bookings from './Bookings'
+import Restaurant from './Restaurant'
+import './App.css'
 
 class App extends Component {
-  render() {
+  render () {
     return (
-      <div className="App">
-        <header className="App-header">CYF Hotel</header>
+      <div className=' App'>
+        <Heading />
+        <div className='Card-row'>
+          <TouristInfoCards
```

Comment:
This component name implies that it renders multiple cards but it actually just renders one. Which isn't really a problem but we can make it more readable.

I would rename your existing component to `TouristInfoCard` (no "s" at the end) and then create a new component named `TouristInfoCards` which then renders the 3 cards that you have here.